modules/core/include/frame.h

```diff
@@ -90,6 +98,8 @@ class Frame{
      */
     void save(std::string dir);
 
+    void undistort(Camera &camera);
```

Comment:
Camera::undistort() and Frame::undistort() both exist. Isn't this a little bit confusing? Which one should be used?